themes/src/main/resources/theme/base/account/account.ftl

```diff
@@ -1,70 +0,0 @@
-<#import "template.ftl" as layout>
```

Comment:
So we're also removing the base?

Reply:
Removing the FreeMarker templates yes. The base messages should stay at least since the account2 relies on those.